tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py

```diff
@@ -309,12 +308,15 @@ def summarize_bug_info(projects):
     for canonical_name, project in projects.items():
         for change in project["changes"]:
             for bug in change["bugs"]:
-                int_id = int(bug["id"])
+                int_id = bug["id"]
```

Comment:
I don't see a point in having `int_id`, no idea what the original author had in mind. Why not always string?

Reply:
I think that could be a veryfication problem or smth.